Dockerfile

```diff
@@ -50,3 +50,6 @@ ENV SENDGRID_PASSWORD=${sendgridPassword}
 
 CMD bundle exec puma --bind tcp://0.0.0.0:8080 --environment production
 EXPOSE 8080
+
+# Create missing folders required by Puma launcher
+RUN MKDIR tmp/pids
```

Comment:
This is only adding this command to the `production` image. You should probably add this around line 24, near the end of the `base` image